<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceWidget_UI</class>
 <widget class="QWidget" name="ResourceWidget_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>457</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="7">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>7</number>
     </property>
     <widget class="QWidget" name="left_box_widget">
      <layout class="QVBoxLayout" name="left_box">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="KLineEdit" name="search_text">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="buildin_box">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="search_results">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="page_number">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="prefix">
             <string>Page </string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="right_box_widget">
      <layout class="QVBoxLayout" name="right_box">
       <item>
        <widget class="QFrame" name="details_box">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextBrowser" name="info_browser">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
            <property name="openLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KUrlLabel" name="label_license">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KUrlLabel" name="provider_info">
            <property name="text">
             <string>Photos provided by Example Provider</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="useCursor">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="KMessageWidget" name="message_line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="closeButtonVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="slider_zoom">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="button_preview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-start">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="button_zoomin">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="zoom-in">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="spacer_horiz">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="button_import">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Import</string>
     </property>
     <property name="icon">
      <iconset theme="document-import">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="button_zoomout">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="zoom-out">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QWidget" name="service_box_2" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="service_box">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Service</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KComboBox" name="service_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>splitter</zorder>
  <zorder>service_box_2</zorder>
  <zorder>message_line</zorder>
  <zorder>button_preview</zorder>
  <zorder>button_import</zorder>
  <zorder>button_zoomin</zorder>
  <zorder>button_zoomout</zorder>
  <zorder>slider_zoom</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QLabel</extends>
   <header>kurllabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>service_list</tabstop>
  <tabstop>button_preview</tabstop>
  <tabstop>button_import</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
